Don't use dprintf when the deployment target is 10.6 or earlier; dprintf
was introduced in 10.7.
--- src/airodump-ng.c.orig	2018-12-09 16:53:44.000000000 -0600
+++ src/airodump-ng.c	2019-12-28 22:48:14.000000000 -0600
@@ -67,6 +67,10 @@
 #include <pcre.h>
 #endif
 
+#ifdef __APPLE__
+#include <AvailabilityMacros.h>
+#endif
+
 #include "version.h"
 #include "pcap.h"
 #include "uniqueiv.h"
@@ -6251,7 +6255,7 @@
 		alarm(1);
 		G.do_exit = 1;
 		signal(SIGALRM, sighandler);
-#if defined(__sun__)
+#if defined(__sun__) || (defined(MAC_OS_X_VERSION_MIN_REQUIRED) && MAC_OS_X_VERSION_MIN_REQUIRED < 1070)
 		fprintf(stdout, "\n");
 #else
 		dprintf(STDOUT_FILENO, "\n");
@@ -6270,7 +6274,7 @@
 
 	if (signum == SIGALRM)
 	{
-#if defined(__sun__)
+#if defined(__sun__) || (defined(MAC_OS_X_VERSION_MIN_REQUIRED) && MAC_OS_X_VERSION_MIN_REQUIRED < 1070)
 		fprintf(stdout,
 #else
 		dprintf(STDERR_FILENO,
